paper.md

```diff
@@ -328,11 +328,56 @@ From Table I & Figure III, it can be seen how the portability of Popper workflow
+These tasks may not be always trivial to accomplish, if we assume the only thing users should care about is writing experimentation scripts and running them inside containers.
+Few examples of this category are stable and mature scientific workflow engines like Nextflow, Pegasus and Taverna which have recently introduced native support running steps in software containers;
+Industry standard workflow engines like Apache Airflow and Luigi [@_luigi_] which require specifying workflows using programming languages and also provide pluggable interfaces that require the installation of separate plugins;
+For example, Airflow and Luigi uses Python, Copper use Java, Dagr use Scala and SciPipe use Go as their workflow definition language.
+The goal with Popper is to minimize overhead both in terms of workflow language syntax and infrastructural requirements for running workflows and hence allow users to focus solely on writing the workflows.
+The first issue is already addressed in the previous subsection, but it's also relevant here because not all engines support standard workflow languages such as CWL and also learning specific programming languages for the purpose of workflow execution seems like an exaggeration.
+Most of these standard workflow engines like Pegasus, Airflow and Luigi also require a standalone service that users need to learn how to deploy and interact with before executing workflows, thus adding to the complexity.
+Popper also mitigates this issue as it can be downloaded and run as a standalone executable and does not assume any service deployment or infrastructural management prior to running workflows.
+
+### Cloud native workflow execution engines
+
+Cloud native computing is an emerging paradigm in software development that aims to utilize existing cloud infrastructure to build, run and scale applications in the cloud.
+Container engines like Docker and Orchestration tools like Kubernetes have become an integral part of the cloud native paradigm over the years.
+With the wide-spread acceptance of the cloud native paradigm, several cloud native workflow engines like  Argo, Pachyderm, Brigade have come into existence.
+These workflow engines facilitate running workflows in the cloud by running steps of a workflow inside containers managed by Kubernetes.
+The limitation of these workflow engines is the requirement of having access to a Kubernetes cluster which can block users from running workflows in the absence of such a Kubernetes cluster.
+Although Popper can run workflows in the cloud using Kubernetes, it does not necessarily require the access to a Kubernetes cluster for running the containerized steps of a workflow.
+With Popper one can run workflows inside containers on the the local machine or on any cloud infrastructure.
+Hence we can say that Popper is not a cloud-native workflow engine, since it does not assume the presence of a cloud environment in order to run workflows.
```

Comment:
Swapping the sections would also help rephrasing this: "Popper is not exclusively cloud-native" . That is, in addition to being able to work as a "cloud-native" (being able to launch a workflow on the cloud), it also allows to be container-native elsewhere (local, slurm, ssh-only, etc.)